src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -886,6 +914,76 @@ GrpAb: Z
     return codomain(map_from_torusinvariant_cartier_divisor_group_to_picard_group(v))
 end
 
+@doc raw"""
+    map_from_picard_group_to_class_group(v::AbstractNormalToricVariety)
+
+Return the embedding of the Picard group into the class group of an abstract normal toric variety `v`.
+
+# Examples
+```jldoctest
+julia> p2 = projective_space(NormalToricVariety, 2)
+Normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric variety without torusfactor
+
+julia> map_from_picard_group_to_class_group(p2)
+Map with following data
+Domain:
+=======
+Abelian group with structure: Z
+Codomain:
+=========
+Abelian group with structure: Z
+```
+"""
+@attr GrpAbFinGenMap function map_from_picard_group_to_class_group(v::AbstractNormalToricVariety)
+    f = image(map_from_torusinvariant_cartier_divisor_group_to_class_group(v))[2]
+    g = snf(domain(f))[2] * f
+    return hom(picard_group(v), class_group(v), matrix(g))
+end
+
+
+############################
+# Gorenstein and Picard Index
+############################
+
+@doc raw"""
+    gorenstein_index(v::AbstractNormalToricVariety)
+
+Return the Gorenstein index of a $\mathbb{Q}$-Gorenstein normal toric variety `v`. 
+This is the smallest positive integer $l$ such that $lK$ is Cartier, where $K$
+is a canonical divisor on `v`.
```

Comment:
If I read exercise 8.3.10 in CLS correct, then the condition is that `-l * K` is Cartier. Also, I would suggest we link to this example in CLS so that it is clear where this index is defined/what is known (according to CLS). Proposal:
```suggestion
Return the Gorenstein index of a $\mathbb{Q}$-Gorenstein normal toric variety `v`. 
This is the smallest positive integer $l$ such that $-l K$ is Cartier, where $K$
is a canonical divisor on `v`. See exercise 8.3.10 and 8.3.11 in [CLS11](@cite) for more details.
```